### ISSUE TITLE
Custom keybinding in new version (0.9.39 beta) not work

### ISSUE THREAD

**Issue by iNarcissuss** [typora/typora-issues#1096]
I added three custom keybings long time ago, but they don't work after I updated to the version 0.9.39(beta)

Any ideas?

OS: window 7 64bit professsion (Chinese version)
Typora version: 0.9.39(beta)

The setting in my configure is as follows
```
  // Custom key binding, which will override the default ones.
  "keyBinding": {
    // for example: 
    // "Always on Top": "Ctrl+Shift+P"
    "Ordered List" : "Ctrl+Alt+O",
	"Unordered List" : "Ctrl+Alt+U",
	"Task List" : "Ctrl+Alt+T"
  },
```

**Comment by louisc** [comment]
I think I can confirm same: 

OS: Windows 10 Enterprise
Typora Version: 0.9.39(beta)

```
  // Custom key binding, which will override the default ones.
  "keyBinding": {
    // for example: 
    // "Always On Top": "Ctrl+Shift+P"
    "Task List": "Ctrl+Shift+N",
    "Toggle Task Status": "Ctrl+Shift+X"
  },
```

**Comment by abnerlee** [comment]
We will fix this ASAP

**Comment by abnerlee** [comment]
fixed in v0.9.40
